### PR TITLE
Rename WindowPadding to ViewPadding

### PR DESCRIPTION
**Pull request by goderbauer**
Follow-up to https://github.com/flutter/engine/pull/39737.
Part of https://github.com/flutter/flutter/issues/116929.

### CODE REVIEW

File: packages/flutter/lib/fix_data/fix_painting.yaml
Comment (goderbauer) [inline packages/flutter/lib/fix_data/fix_painting.yaml:1]: @Piinks Is there anything else I need to do when writing the first fix for a new library? I also added tests in `packages/flutter/test_fixes/painting/`.

**Review by loic-sharma — APPROVED** [review]
Changes look good! Consider getting a second review from someone that's done a Dart fix before if you have any concerns with the YAML. Otherwise, :shipit:

**Review by Piinks — APPROVED** [review]
LGTM!

**Pull request merged**
